app/controllers/search_controller.rb

```diff
@@ -127,8 +130,26 @@ def feed_content
                 elsif params[:class_name] == "User"
                   # No need to check for articles or podcast episodes if we know we only want users
                   user_search
-                else
-                  # if params[:class_name] == PodcastEpisode, Article, or Comment then skip user lookup
+                elsif params[:class_name] == "Article" && params[:search_fields].blank? # homepage
+                  if FeatureFlag.enabled?(:search_2_homepage)
+                    # NOTE: published_at is sent from the frontend in the following ES-friendly format:
+                    # => {"published_at"=>{"gte"=>"2021-04-06T14:53:23Z"}}
+                    published_at_gte = params.dig(:published_at, :gte)
+                    published_at_gte = Time.zone.parse(published_at_gte) if published_at_gte
+                    published_at = published_at_gte ? published_at_gte.. : nil
```

Comment:
Surprisingly `nil..` works fine in `ActiveRecord` queries, it gets turned into `WHERE 1=1` so technically you don't need this ternary.

Reply:
I'm just trying to not poke the DB bear :D Also, it's technically temporary code. We can't touch the frontend, but once ES will be gone and feature flags are removed, we will, hopefully :D